kernels/etherlink/Makefile

```diff
@@ -52,16 +50,23 @@ cargo-check-docs:
 	@INBOX_FILE=$(INBOX_FILE) cargo doc --all-features --document-private-items --no-deps
 
 cargo-build-kernel:
-	@cargo build --package etherlink --release
-
-cargo-build-kernel-static:
-	@INBOX_FILE=$(INBOX_FILE) cargo build -p etherlink --release --features static-inbox
+	@INBOX_FILE=$(INBOX_FILE) cargo build --package etherlink --release $(if $(FEATURES),--features $(FEATURES))
```

Comment:
You don't define `FEATURES` anywhere. Are you relying on `make` to import this environment variable?

Reply:
Yes. I've made a small change to redefine it if unset [here](https://github.com/tezos/riscv-pvm/compare/68b92c020ace147306cef3118066ba5a85449f32..49ccb19efc4232afb4edc43bd62d471249d64708).

Reply:
Done [here](https://github.com/tezos/riscv-pvm/compare/49ccb19efc4232afb4edc43bd62d471249d64708..fc09ca9ae8d1f8557f98d8bac69891e368e82cdc).